packages/tuareg/tuareg.2.0.8/opam

```diff
@@ -1,20 +1,15 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "Christophe.Troestler@umons.ac.be"
-build: [
-  [make "install" "INSTALL_DIR=%{prefix}%/share/tuareg"]
+authors: [
+  "Albert Cohen <Albert.Cohen@prism.uvsq.fr>"
+  "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
 ]
-remove: [
-  [make "uninstall"]
-]
-depends: []
-depexts: [
- [["ubuntu"] ["emacs"]]
- [["debian"] ["emacs"]]
-]
-post-messages: [
-"Please add in ~/.emacs.d/init.el or ~/.emacs to following lines:
-    (add-to-list 'load-path \"INSTALL_DIR\")
-    (load \"tuareg-site-file\")
-where you must replace INSTALL_DIR by the value displayed during the
-above installation"
-  {success} ]
+homepage: "https://forge.ocamlcore.org/projects/tuareg/"
+bug-reports: "https://forge.ocamlcore.org/tracker/?group_id=43"
+dev-repo: "https://github.com/ocaml/tuareg.git"
+build: [make "tuareg-site-file.el"]
+post-messages: "
+If you have not yet done so, please add in ~/.emacs.d/init.el or
+in ~/.emacs to following lines:
+    (add-to-list 'load-path \"%{share}%/emacs/site-lisp\")
```

Comment:
I think `(load \"%{share}%/emacs/site-lisp/tuareg-site-file\")` would be better — it will add the correct location to the path.